examples/README.md

```diff
@@ -186,3 +186,32 @@ kubectl port-forward jupyter-notebook-server 8888:8888
 # clean up
 kubectl delete -f ./examples/jupyter/jupyter-notebook-server.yaml
 ```
+
+## Jupyter Notebook Example (no experimental read cache)
```

Comment:
Do you mean "(with experimental read cache)"?

Reply:
Fixed! Thanks